app/assets/stylesheets/frontend/base.scss

```diff
@@ -142,8 +142,10 @@ $govuk-typography-use-rem: false;
 // When removing ensure all cases are captured - some rules in helpers/_headings.scss
 // are very specific, requiring the !important here
 #whitehall-wrapper {
-  a.govuk-link{
+  a.govuk-link,
+  a[class^="gem-c-"] {
```

Comment:
I'm a bit confused: I'm not sure this matches your commit message...?

Reply:
The attribute selector means that components with links that don't have the `govuk-link` class don't inherit the focus styles from Whitehall, as they're included in the override. It's mentioned in the message for 5cf93bb48e16076d26e8c484b26f2d74062ad139.

Reply:
Ah sorry I completely mis-read it.